docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
       dockerfile: ./Dockerfile
 
   nlp_api:
-    restart: always
+    # restart: always
```

Comment:
is this change ok ?

Reply:
woups I didn't mean to commit this 🙄 I'm doing this locally because I don't know how to set it up and so it keeps restarting